spring-integration-hazelcast/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+group = 'org.springframework.integration'
+
+description = 'Spring Integration Hazelcast Support'
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+
+
+repositories {
+        
+     maven { url "http://repo.maven.apache.org/maven2" }
+}
+
+dependencies {
+    compile group: 'org.springframework', name: 'spring-context', version:'4.1.5.RELEASE'
+    compile group: 'org.springframework.integration', name: 'spring-integration-core', version:'4.1.2.RELEASE'
+    compile group: 'com.hazelcast', name: 'hazelcast', version:'3.4.1'
+    compile group: 'com.hazelcast', name: 'hazelcast-spring', version:'3.4.1'
```

Comment:
No sure that we need this dep as `compile`. 
As `testCompile` it is good, although it is even `testRuntime`